packages/app/pages/Registration/organizationRegistration.tsx

```diff
@@ -57,6 +148,10 @@ class OrganizationTypeSection extends React.Component {
  */
 class OrganizationNameSection extends React.Component {
 
+  private state = {
+    nameValid: true
```

Comment:
Because if we don't, it starts the page by yelling at the user before they have a chance to type in their name. For the name, anything is valid as long as it exists. By assuming it is valid from the beginning, as soon as they start typing it stays valid. If they remove their name, it becomes invalid then. If they attempt to submit it without a name it still flags the submission as invalid